packages/GenderSelection/src/GenderSelection.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState, AllHTMLAttributes, FC, RefObject, RefAttributes, useRef, useEffect} from 'react';
+import classnames from 'classnames';
+import styles from './GenderSelection.scss';
+import { Input } from "@mbkit/input";
+import { Select } from "@mbkit/select";
+import { Label } from "@mbkit/label";
+
```

Comment:
Could you add `@mbkit/input`, `@mbkit/select`, and `@mbkit/label` as dependencies?

Reply:
Added as a dependency